### PR TITLE
[MBL-220] manage egress configuration through worker flags 

### PR DESCRIPTION
**Pull request by joanxie878**
What this PR do?

We want to control uptime egress through worker flags. In this case, we use two worker flag, "egress_test_size" for the size of the test array, "egress_test_frequency" for how many times to send the array per tick.

Test Link:(below test are based on 3*3 grid for 50 players)

https://buildkite.com/improbable/unrealgdk-nfr/builds/4080   (for 1size * 1frequency)
https://buildkite.com/improbable/unrealgdk-nfr/builds/4086    (for 2size * 5frequency)

Test Result:

1size*1frequiency:
image

2size*5frequency:
image

### CODE REVIEW

File: Game/Source/GDKTestGyms/AUptimeGameMode.cpp [hunk 1]
```diff
@@ -33,6 +35,8 @@ AUptimeGameMode::AUptimeGameMode()
 	, SpawnRows(1)
 	, ZoneWidth(1000000.0f)
 	, ZoneHeight(1000000.0f)
+	, TestDataSize(0)
+    , TestDataFrequency(0)
```
Comment (jayprobable) [inline Game/Source/GDKTestGyms/AUptimeGameMode.cpp:39]: whitespace

File: Game/Source/GDKTestGyms/AUptimeGymsCharacter.cpp [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	}
+}
+
+void AUptimeGymsCharacter::EgressTestForRPC()
+{
+	if (HasAuthority())
+	{
+		RPCsForAuthority();
+	}
+	else
```
Comment (jayprobable) [inline Game/Source/GDKTestGyms/AUptimeGymsCharacter.cpp:54]: I don't think we need to send client to server RPCs

**Review by jayprobable — CHANGES_REQUESTED** [review]
Let's remove the ingress, client to server RPCs and just keep the outbound.. server to client.

File: Game/Source/GDKTestGyms/AUptimeGymsCharacter.cpp (hunk 2 above)
Comment (joanxie878) [reply Game/Source/GDKTestGyms/AUptimeGymsCharacter.cpp:54]: ok, I will remove the client to server RPCs

**Review by jayprobable — APPROVED** [review]

**Review by siliviali — APPROVED** [review]

**Pull request merged**